play-java-starter-example/app/views/welcome.scala.html

```diff
@@ -51,11 +55,11 @@ <h2>Why do you see this page?</h2>
                 The template is defined in the <code>app/views/index.scala.html</code> file and compiled as a standard Java class.
             </p>
 
-            <pre><code>@@(message: String)
+            <pre><code>@@(message: String)(implicit assetsFinder: AssetsFinder)
 
   @@main("Welcome to Play") {
 
-  @@play20.welcome(message, style = "Java")
+  @@welcome(message, style = "Java")
```

Comment:
While doing applies changes after the PR comments, I've noticed `welcome.scala.html` had yet another copy of the code. I've raised https://github.com/playframework/play-samples/issues/84 to tackle this never-ending duplication.